Sources/Client/Client/Client+User.swift

```diff
@@ -25,7 +41,11 @@ public extension Client {
                     pagination: Pagination = [.usersPageSize],
                     options: QueryOptions = [],
                     _ completion: @escaping Client.Completion<[User]>) -> Cancellable {
-        queryUsers(query: .init(filter: filter, sort: sort, pagination: pagination, options: options), completion)
+        if let sort = sort {
+            return queryUsers(query: .init(filter: filter, sorting: [sort], pagination: pagination, options: options), completion)
+        } else {
+            return queryUsers(query: .init(filter: filter, sorting: [], pagination: pagination, options: options), completion)
+        }
```

Comment:
```suggestion
queryUsers(query: .init(filter: filter, sorting: (sort == nil ? [] : [sort!]),  pagination: pagination,  options: options), 
           completion)
```

Reply:
Actually that was my first code, but I hate force unwrapping in production even when it's 100% safe so I went this route. A bit more code but 1000% safe (irrationally so)

Reply:
Force unwrap is always a warning for developers. In this case it's pretty clear because it's small case `sort == nil ? [] : [sort!]`